editor/editor_vcs_interface.h

```diff
@@ -51,16 +51,25 @@ class EditorVCSInterface : public Object {
 	bool _is_vcs_initialized();
 	Dictionary _get_modified_files_data();
 	void _stage_file(String p_file_path);
+	void _discard_file(String p_file_path);
 	void _unstage_file(String p_file_path);
 	void _commit(String p_msg);
 	Array _get_file_diff(String p_file_path);
 	bool _shut_down();
 	String _get_project_name();
 	String _get_vcs_name();
+	Array _get_previous_commits();
+	Array _get_branch_list();
+	void _pull();
+	void _push();
+	void _fetch();
+	bool _checkout_branch(String branch);
+	void _set_up_credentials(String username, String password);
 
 public:
 	static EditorVCSInterface *get_singleton();
 	static void set_singleton(EditorVCSInterface *p_singleton);
+	int max_commit_fetch = 10;
```

Comment:
I think this is being used as a constant. You could declare this as const or change this to a macro, so this is not modified by anyone